### PR TITLE
ip6 tunnel: add support for DSCP

### PR DESCRIPTION
**Pull request by ssahani**

### CODE REVIEW

File: man/systemd.netdev.xml [hunk 1]
```diff
@@ -504,6 +504,15 @@
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><varname>DiffServiceCodePoint=</varname></term>
+        <listitem>
+          <para>A boolean. When true, inherits Differentiated
+          Service Code Point (DSCP) field between inner and outer
+          header for ip6_tunnel.</para>
```
Comment (zonque) [inline man/systemd.netdev.xml]: inherits _the_ ...

Also, could you give a little more background here on what the effect of this flag is?

Comment (ssahani) [inline man/systemd.netdev.xml]: It's a part of ipv6 header,  used for QOS.

Comment (zonque) [inline man/systemd.netdev.xml]: I meant to actually put that information in the man page, along with the missing 'the'.

Comment (ssahani) [inline man/systemd.netdev.xml]: ohh it's like definition of DSCP here

**Comment by ssahani** [comment]
updated.

**Pull request merged**